4_processing_features/extract_single_cells.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# CellProfiler methods
```

Comment:
Consider rewriting this comment and the other in this file to clarify what these nbconvert commands do.

Reply:
Fixed in commit [e2512de](https://github.com/WayScience/NF1_SchwannCell_data/pull/42/commits/e2512de1d4f51232a14dc02f74361e09342550a0)